app/presenters/notices/setup/abstraction-alerts-notifications.presenter.js

```diff
@@ -0,0 +1,121 @@
+'use strict'
+
+/**
+ * Formats recipients into notifications for an abstraction alert
+ * @module AbstractionAlertsNotificationsPresenter
+ */
+
+const { transformStringOfLicencesToArray, timestampForPostgres } = require('../../../lib/general.lib.js')
+const { notifyTemplates } = require('../../../lib/notify-templates.lib.js')
+const { contactName, contactAddress } = require('../../crm.presenter.js')
+
+/**
+ *
+ * @param recipients
+ * @param session
+ * @param eventId
+ */
+function go(recipients, session, eventId) {
+  const notifications = []
+
+  for (const recipient of recipients) {
+    // loop each threshold we need to use ?
```

Comment:
We need to iterate over the 'relevant' licence monitoring stations to send the notification to the recipient per station.